PermissionsService/Models/PathPermissions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PermissionsService.Models
+{
+    internal class PathPermissions
+    {
+        public string Url
```

Comment:
```suggestion
        public required string Url
```

(this way you can get rid of the constructor)